vms/platformvm/txs/executor/import_test.go

```diff
@@ -180,7 +186,7 @@ func fundedSharedMemory(
 		utxo := &avax.UTXO{
```

Comment:
Should we remove the `// #nosec G404` here?

Reply:
removed.